sql/core/src/test/resources/structured-streaming/file-sink-log-version-2.1.0/8

```diff
@@ -1,3 +1,2 @@
 v1
 {"path":"/a/b/8","size":800,"isDir":false,"modificationTime":800,"blockReplication":1,"blockSize":100,"action":"add"}
-{"path":"/a/b/0","size":100,"isDir":false,"modificationTime":100,"blockReplication":1,"blockSize":100,"action":"delete"}
```

Comment:
This should be removed - this gave false information as we support DELETE_ACTION previously, but in reality it was never implemented.